packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -571,6 +571,11 @@ class FlutterDriver {
     return result.changedState;
   }
 
+  Future<int> getSemanticsId(SerializableFinder finder, { Duration timeout = _kShortTimeout}) async {
```

Comment:
doc comment?

Reply:
Addressed in https://github.com/flutter/flutter/pull/19047